.bazelrc

```diff
@@ -192,3 +192,8 @@ build:ubsan --linkopt -fno-sanitize-recover=all
 # Import local specific llvm config options, which can be generated by
 # ci/env/install-llvm-dependencies.sh
 try-import %workspace%/.llvm-local.bazelrc
+
+# Even with sandbox mode bazel prioritizes system headers over the ones in the sandbox.
+# It picks up the system headers when someone has protobuf installed via Homebrew.
+# Work around for https://github.com/bazelbuild/bazel/issues/8053
+build:macos --sandbox_block_path=/usr/local/
```

Comment:
Not sure whether it's something needed for everyone. Should you try to just put it into your global bazel config locally?

Reply:
it's from the upstream